src/lib/select/select.spec.ts

```diff
@@ -1374,10 +1374,26 @@ describe('MatSelect', () => {
 
         fixture.componentInstance.foods[1].viewValue = 'Calzone';
         fixture.detectChanges();
+        tick();
```

Comment:
Note: this is only necessary for our assertions to pass, it won't be necessary for people to change their existing tests.